openedx/features/course_experience/static/course_experience/js/CourseOutline.js

```diff
@@ -0,0 +1,37 @@
+/* globals Logger */
```

Comment:
Couldn't import logger from `common/static/js/src/logger.js` because it's not a real AMD module. However, the existing Require build process puts it onto `window`, so we can just use it as a global. In the future I'd like to write a loader that can fix issues like these in our older modules so all of our webpack code can be statically analyzable and there are no hidden dependencies.